controller/work_item_events_test.go

```diff
@@ -186,18 +187,25 @@ func (s *TestEvent) TestListEvent() {
 	})
 
 	s.T().Run("event list ok - description", func(t *testing.T) {
-		fxt := tf.NewTestFixture(s.T(), s.DB, tf.CreateWorkItemEnvironment(), tf.WorkItems(1))
+		fxt := tf.NewTestFixture(t, s.DB, tf.CreateWorkItemEnvironment(), tf.WorkItems(1))
 		svc := testsupport.ServiceAsSpaceUser("Event-Service", *fxt.Identities[0], &TestSpaceAuthzService{*fxt.Identities[0], ""})
 		EventCtrl := NewEventsController(svc, s.GormDB, s.Configuration)
 		workitemCtrl := NewWorkitemController(svc, s.GormDB, s.Configuration)
 		spaceSelfURL := rest.AbsoluteURL(&http.Request{Host: "api.service.domain.org"}, app.SpaceHref(fxt.Spaces[0].ID.String()))
+
+		modifiedDescription := "# Description is modified1"
+		modifiedRenderedDescription := "<h1>Description is modified1</h1>\n"
```

Comment:
The update controller would convert the plaintext `modifiedDescription` to `rendered description`. Why you want to add it?

Reply:
True. I've changed it here: 02671899ea68ba4926b5cbb419b17cae360671f7